Gopkg.toml

```diff
@@ -83,5 +87,9 @@ ignored = ["github.com/fnproject/fn/cli",
   revision = "0ae9b6f38f24f65567d4b46602502b33c95cf57a"
 
 [[constraint]]
-    name = "github.com/prometheus/client_golang"
-    version = "0.9.0-pre1"
+  name = "github.com/prometheus/client_golang"
+  version = ">=0.9.0"
+
+[[constraint]]
+  name = "golang.org/x/net"
+  branch = "master"
```

Comment:
this one wasn't getting updated when running `dep ensure -update` ?

Reply:
I just wanted to be explicit. Since thats what we use.

Reply:
can you remove please? as few constraints as possible helps, dep will handle it until it's an issue

Reply:
K